index.html

```diff
@@ -174,5 +175,37 @@ <h1>I'm Shimels, Glad to see you!</h1>
         </li>
       </ul>
     </section>
+    <section id="contact-form">
+      <h1>Contact me</h1>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form action="https://formspree.io/f/xyyqzzjg" method="post">
+        <input
+          type="text"
+          id="name"
+          name="user_name"
+          placeholder="Write your name here"
+          maxlength="30"
+          required
+        />
+        <input
+          type="email"
+          id="email"
+          name="user_email"
+          placeholder="Write your email here"
+          required
+        />
+        <textarea
+          name="user_message"
+          placeholder="Write your message here"
+          maxlength="500"
+          required
+        ></textarea>
+        <button type="submit" class="content-submit-btn">Get in touch</button>
+      </form>
+    </section>
```

Comment:
👍👏 Great job on adding the contact us section! 📝📞 However, I noticed there's some spacing around the section on the left, right, and bottom. As per the design, there shouldn't be any unnecessary white gaps. 🚫⚪ Please consider removing the spacing and updating your code to match the design as closely as possible. 🛠️🎨

| Your | Professional |
| --- | --- |
| ![image](https://github.com/shimels1/portfolio_microverse/assets/55840999/63c6f439-485b-47d4-8b9b-3d14cb3baa9d) | ![image](https://github.com/shimels1/portfolio_microverse/assets/55840999/78b0113a-b04c-4db3-b033-774436f7b142) |

By ensuring a seamless integration with the design, your contact us section will look even more polished and professional. Keep up the good work, and I'm excited to see the final result! 🌟😊